bin/dev/em-forward.sh

```diff
@@ -54,8 +54,8 @@ freeview -v ../mri/T1.mgz -f watershed/*-low -viewport coronal
 for surf in *_surface-low
 do
     python<<EOF
-import bnm.recon.io.surf
-bnm.recon.io.surf.convert_fs_to_brain_visa("$surf")
+import bnm.recon.algo.reconutils
```

Comment:
This could be shorter written:
```
from bnm.recon.algo import reconutils
reconutils.convert_fs_to_brain_visa("$surf")
```
or even better:
```
from bnm.recon.algo.reconutils import convert_fs_to_brain_visa
convert_fs_to_brain_visa("$surf")
```
The advantage would be that of the function moves into a new module, only one of the lines will require update, not both.